interop-tests/Cargo.toml

```diff
@@ -7,14 +7,15 @@ publish = false
 [dependencies]
 anyhow = "1"
 async-trait = "0.1.58"
+either = "1.8.0"
 env_logger = "0.9.0"
 futures = "0.3.1"
 if-addrs = "0.7.0"
 log = "0.4"
+rand = "0.8.5"
 redis = { version = "0.22.1", features = ["tokio-native-tls-comp", "tokio-comp"] }
+strum = { version = "0.24.1", features = ["derive"] }
 tokio = { version = "1.24.1", features = ["full"] }
 
 libp2p = { path = "../", default_features = false, features = ["websocket", "quic",  "mplex", "yamux", "tcp", "tokio", "ping", "noise", "tls", "dns", "rsa", "macros", "webrtc"] }
-rand = "0.8.5"
-strum = { version = "0.24.1", features = ["derive"] }
-either = "1.8.0"
+
```

Comment:
>[thomaseizinger](https://github.com/thomaseizinger) [14 hours ago](https://github.com/libp2p/rust-libp2p/pull/3331#discussion_r1081904620)
Any reason for this newline?

was to separate libp2p from the other dependencies, meanwhile with `cargo add` that was lost, separated again with sorting them ptal.

Reply:
> >[thomaseizinger](https://github.com/thomaseizinger) [14 hours ago](https://github.com/libp2p/rust-libp2p/pull/3331#discussion_r1081904620)
> Any reason for this newline?
> 
> was to separate libp2p from the other dependencies, meanwhile with `cargo add` that was lost, separated again with sorting them ptal. 

Do we need to separate libp2p? Isn't it just another dependency? :)